packages/web-components/src/pane/pane.template.ts

```diff
@@ -0,0 +1,39 @@
+import { ElementViewTemplate, html, ref } from '@microsoft/fast-element';
+import type { Pane } from './pane.js';
+
+/**
+ * The template for the Pane component.
+ * @public
+ */
+
+export function paneTemplate<T extends Pane>(): ElementViewTemplate<T> {
+  return html<T>`
+    <template
+      role="${x => (x.modal ? 'dialog' : 'complementary')}"
```

Comment:
It would be good to understand the use cases of dialog and complementary - is one an "outer pane" and the other "slotted panes"? What prevents dialogs from being nested in dialogs? I'd like to understand this better because ideally we don't need to reinvent the native dialog to get dialog behavior and semantics...if it's a must, that's fine - but I don't think I have nearly enough clarity to understand what exists and why.